### PR TITLE
disable scroll when modal open

### PR DESCRIPTION
**Pull request by DerrykBoyd**
## What issue is this referencing? #362 

Note: Removed the transition-all from the body tag as it caused a weird bounce effect when switching the overflow property. I clicked around the site and couldn't see any difference from removing that class.

## Do these code changes work locally and have you tested that they fix the issue yourself?

-   [x] yes!

## Does the following command run without warnings or errors?

-   [x] yes! `npm run pr-checks`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [x] yes!

## My node version matches the one suggested when running `nvm use`?

-   [x] yes!

### CODE REVIEW

File: components/app/Modal.vue [hunk 1]
```diff
@@ -10,6 +12,16 @@ const closeModal = () => {
   modal.value = false
 }
 
+watch(modal, (value) => {
```
Comment (arashsheyda) [inline components/app/Modal.vue:15]: it would better to use `:class` with tailwind classes on the modal

File: app.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@ useServerSeoMeta({
 
 <template>
   <Html>
-    <Body class="bg-neutral-100 dark:bg-neutral-900 dark:text-white transition-all duration-300">
```
Comment (arashsheyda) [inline app.vue:13]: the reason we added `transition` here is for switching between dark mode and light mode

Comment (DerrykBoyd) [reply app.vue:13]: Ah, thanks for the context. I've reduced the transition to `transition-colors` which should satisfy both issues.

File: components/app/Modal.vue (hunk 1 above)
Comment (DerrykBoyd) [reply components/app/Modal.vue:15]: These styles are targeted to the `body` element, can you clarify more what you mean?

Comment (arashsheyda) [reply components/app/Modal.vue:15]: Ah yes sorry I thought we are using a different approach, this works! thanks

**Review by arashsheyda — APPROVED** [review]

**Pull request merged**